lib/view/ui/userdatawizard/personalinfo_page.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:community/view_model/userwizard/wizard_controller.dart';
+import 'package:community/utils/app_string_res.dart';
+import 'package:community/utils/dimen.dart';
+import 'package:community/view_model/userwizard/personalinfo_controller.dart';
+
+
+
+class PersonalInfoPage extends StatelessWidget {
+  final PersonalInfoController controller = Get.put(PersonalInfoController());
+  final UserWizardController wizardController = Get.put(UserWizardController());
+
+  PersonalInfoPage({super.key});
+
+  final formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text(personalinfomessage),
+        backgroundColor: Colors.blueAccent,
+      ),
+      body: Form(
+        key: controller.formKey, // Add a form key
+        child: Column(
+          children: [
+            ElevatedButton(
+              onPressed: controller.pickImage,
+              child: const Text(imagepickertext),
+            ),
+            const SizedBox(height: Dimen_20),
+            Obx(() {
+              return controller.selectedImage.value != null
+                  ? Image.file(controller.selectedImage.value!)
+                  : Container();
+            }),
+            TextFormField(
+              key: Key('firstNameField'),
+              controller: controller.firstNameController,
+              decoration: const InputDecoration(labelText: firstname),
+              validator: (value) {
+                if (value!.isEmpty ||!RegExp(r'^[a-z A-Z]').hasMatch(value!)) {
```

Comment:
make validator class global, Suppose you have to validate email of 5 diffferent places on app.